docs/user-guides/upgrade-cluster.md

```diff
@@ -0,0 +1,137 @@
+# Upgrade Vald cluster
+
+Before this document, we recommend reading [the configuration document](../user-guides/configuration.md) first.
+
+## Update Configuration
+
+When you'd like to update the configuration for the Vald cluster, you need to edit your configuration file and apply it according to your deployment method.
+Please refer to one of the sections below according to your deployment method.
+
+<div class="caution">
+If each Vald Agent already has the indexes and uses the backup feature, please DO NOT edit the dimension and the distance type to avoid initContainer error.<BR>
+(Each Vald Agent will restore from backup data when backup when performing a rolling update)
+</div>
+
+<div class="warn">
+The indexes will be clear when performing a rolling update if each Vald Agent already has the indexes and does not use the backup.
+</div>
+
+### Using Helm for deployment
+
+The configuration file is `values.yaml`. (https://helm.sh/docs/chart_template_guide/values_files/)
+
+
+You can edit your `values.yaml` and update your Vald cluster by following steps:
+1. Edit `values.yaml`.
+1. Update with Helm command. 
+
+    ```bash
+    helm upgrade <NAME> vald/vald --values <YOUR VALUES FILE PATH>
+    # e.g., helm upgrade vald vald/vald --values valeus.yaml
+    ```
```

Comment:
I think it might be better to ask to confirm the helm release name before upgrading it, what do you think?
```suggestion
    ```bash
    # List helm release name
    helm list
    
    helm upgrade <NAME> vald/vald --values <YOUR VALUES FILE PATH>
    # e.g., helm upgrade vald vald/vald --values valeus.yaml
    ```
```

Reply:
It is no way to check current version in own repo